test-programs/account-compression-test/tests/address_merkle_tree_tests.rs

```diff
@@ -1288,29 +1339,29 @@ async fn test_address_merkle_tree_and_queue_rollover_default() {
 
 #[tokio::test]
 async fn test_address_merkle_tree_and_queue_rollover_custom() {
-    for changelog_size in (1000..5000).step_by(1000) {
-        for queue_capacity in [5003, 6857, 7901] {
-            let roots_size = changelog_size * 2;
-
-            for address_changelog_size in (250..1000).step_by(250) {
-                address_merkle_tree_and_queue_rollover(
-                    &AddressMerkleTreeConfig {
-                        height: ADDRESS_MERKLE_TREE_HEIGHT as u32,
-                        changelog_size,
-                        roots_size,
-                        canopy_depth: ADDRESS_MERKLE_TREE_CANOPY_DEPTH,
-                        address_changelog_size,
-                        network_fee: Some(5000),
-                        rollover_threshold: Some(95),
-                        close_threshold: None,
-                    },
-                    &AddressQueueConfig {
-                        capacity: queue_capacity,
-                        sequence_threshold: roots_size + SAFETY_MARGIN,
-                        network_fee: None,
-                    },
-                )
-                .await;
+    for changelog_size in (1..=5000).step_by(1000) {
+        for roots_size in (changelog_size..=5000).step_by(1000) {
+            for queue_capacity in [5003, 6857, 7901] {
+                for address_changelog_size in (250..1000).step_by(250) {
+                    address_merkle_tree_and_queue_rollover(
+                        &AddressMerkleTreeConfig {
+                            height: ADDRESS_MERKLE_TREE_HEIGHT as u32,
+                            changelog_size,
+                            roots_size,
+                            canopy_depth: ADDRESS_MERKLE_TREE_CANOPY_DEPTH,
+                            address_changelog_size,
+                            network_fee: Some(5000),
+                            rollover_threshold: Some(95),
```

Comment:
Iterations make sense!
We can just put the rollover threshold to zero to test this one, that should be faster.
Imo we should still keep one test that advances the slot and modifies the account (idk whether these replaced that original test which did that.)